modules/hdf5_handler/HDF5CF.h

```diff
@@ -798,7 +806,7 @@ class File {
     File(const char *h5_path, hid_t file_id) :
         path(std::string(h5_path)), fileid(file_id), rootid(-1), unsupported_var_dtype(false), unsupported_attr_dtype(false), unsupported_var_dspace(
             false), unsupported_attr_dspace(false), unsupported_var_attr_dspace(false), addeddimindex(0), check_ignored(
-            false), have_ignored(false), have_udim(false),_is_dap4(false)
+            false), have_ignored(false), have_udim(false),_is_dap4(false),iscoard(false)
```

Comment:
There is a feature in C++-11 that can be used to simplify writing these constructors where fields like 'iscoard' can be initialized where they are defined in the header and don't need to be included in the constructors. The notion is that those initial values don't have to be repeated for each constructor. The SonarScan will include this under what it calls a 'code smell.'
There's no need to change this now, but it's something to keep in mind that might help writing new code easier.

Reply:
Thanks, will handle this with another ticket.